signer/src/stacks/api.rs

```diff
@@ -150,6 +152,34 @@ pub enum SubmitTxResponse {
     Rejection(TxRejection),
 }
 
+/// The account info for a stacks account.
+pub struct AccountInfo {
+    /// The balance of the account in micro-STX
+    pub balance: u128,
+    /// The amount locked (stacked?) in micro-STX
+    pub locked: u128,
+    /// The last known nonce of the account.
+    pub nonce: u64,
+}
+
+/// Helper function for converting a hexidecimal string into an integer.
+fn parse_hex_u128(hex: &str) -> Result<u128, Error> {
+    let hex_str = hex.strip_prefix("0x").unwrap_or(hex);
```

Comment:
Nit: We can skip the extra unwrap logic by using `trim_start_matches`.
```suggestion
    let hex_str = hex.trim_start_matches("0x");
```

Reply:
Oh, perfect! Thank you.